students/itsabhi/Lesson4/mailroom2.py

```diff
@@ -0,0 +1,69 @@
+#! /usr/bin/env python3
+
+def sending_thanksv2(donor_hist):
+
+    donor = input("Type the Full Name of the donor here or type List to display current donors : ")
```

Comment:
I would suggest placing your user input prompts in while loops so you don't have to repeat them in your conditional code blocks below